app/views/users/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Users#index</h1>
```

Comment:
- [ ] You have already added the controller actions for showing all users, but maybe you forget the handle the views -- let's add the views for index users.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone3/userctr.png)


- [ ] **[OPTIONAL]** Consider using [`patials`](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) for users to DRY your views.

Reply:
We have not yet implemented it because we have not yet reached the friendship milestone